src/v2i-hub/CDASimAdapter/src/include/CDASimConnection.hpp

```diff
@@ -142,6 +163,9 @@ namespace CDASimAdapter {
             uint _v2x_port;
             tmx::utils::Point _location;
             bool _connected = false;
+            std::string _sensor_json_file_path;
+            //Global variable to store the sensors information
+            Json::Value _sensors_json_v;
```

Comment:
Why not make a sensor class

Reply:
I think Json::Value is sufficient for this. Do you have specific reason that we need a sensor class?